framework/tests/returnsreceiverlombok/lombok.config

```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
@kelloggm it seems that if Lombok is run without this option, the Returns Receiver Checker does not add annotations properly, since it looks for `@lombok.Generated` annotations.  I'm guessing the same is true for Object Construction Checker.  Does this sound right?  If so, we should document it somewhere.

Reply:
That's correct. I suspect the OCC's documentation doesn't mention it because it requires users to use the Gradle plugin, which does this automatically.

I'll still make a PR against the OCC to add a note about it.